src/WebPush.php

```diff
@@ -204,11 +212,36 @@ private function prepareAndSend(array $notifications)
                 $headers['Topic'] = $options['topic'];
             }
 
-            if (substr($endpoint, 0, strlen(self::GCM_URL)) === self::GCM_URL) {
-                if (array_key_exists('GCM', $this->apiKeys)) {
-                    $headers['Authorization'] = 'key='.$this->apiKeys['GCM'];
+            // if GCM or FCM url
+            $isGCM = substr($endpoint, 0, strlen(self::GCM_URL)) === self::GCM_URL;
+            if ($isGCM || substr($endpoint, 0, strlen(self::FCM_URL)) === self::FCM_URL) {
```

Comment:
GCM is the only endpoint that will require the API key in the authorization header.

FCM is just a normal web push endpoint and will only be returned when the developer is using VAPID's public key in the subscribe() call.